crates/rpc/rpc-eth-types/src/error.rs

```diff
@@ -494,7 +503,7 @@ impl From<reth_primitives::InvalidTransactionError> for RpcInvalidTransactionErr
         // txpool (e.g. `eth_sendRawTransaction`) to their corresponding RPC
         match err {
             InvalidTransactionError::InsufficientFunds { .. } => Self::InsufficientFunds,
-            InvalidTransactionError::NonceNotConsistent => Self::NonceTooLow,
+            InvalidTransactionError::NonceNotConsistent => Self::NonceTooLow { tx: 0, state: 0},
```

Comment:
I think we can also add these fields to the `NonceNotConsistent` error variant

https://github.com/paradigmxyz/reth/blob/29058ad7ee64d993de6c301a9d688002cb698f03/crates/transaction-pool/src/validate/eth.rs#L338-L344